spec/features/admin_split_decision_create_spec.rb

```diff
@@ -36,6 +36,12 @@
     end
 
     expect(split_page).to be_displayed
+    expect(split_page).to have_content "Queued decision"
+
+    Delayed::Worker.new.work_off
+
+    split_page.load split_id: split.id
+    expect(split_page).to be_displayed
 
     expect(Assignment.where(variant: "hammer_time").count).to eq 0
     expect(Assignment.where(variant: "touch_this").count).to eq 5
```

Comment:
I guess this is using truncation strategy being a feature spec, and yeah, it works by default because that's how DatabaseCleaner works unless you exclude the table, eh? :)